parachain/runtime/kintsugi/src/lib.rs

```diff
@@ -1181,6 +1233,9 @@ construct_runtime! {
         VaultStaking: staking::{Pallet, Storage, Event<T>} = 42,
         VaultCapacity: reward::<Instance3>::{Pallet, Storage, Event<T>} = 43,
 
+        Farming: farming::{Pallet, Call, Storage, Event<T>} = 44,
```

Comment:
Should we move the farming pallet to the DeFi pallets in the 1xx range? No strong opinion on this though.